sys/include/net/conn/udp.h

```diff
@@ -105,9 +105,9 @@ int conn_udp_recvfrom(conn_udp_t *conn, void *data, size_t max_len, void *addr,
 /**
  * @brief   Sends a UDP message
  *
- * @param[in] data      Pointer where the received data should be stored.
- * @param[in] len       Maximum space available at @p data.
- * @param[in] src       The source address. May be NULL for all any interface address.
+ * @param[in] data      Pointer to the payload.
```

Comment:
What's "the payload" in this context? I'm asking from a perspective of a person that does not know much about UDP.